templates/crds/addressspaceplans.crd.yaml

```diff
@@ -43,6 +44,13 @@ spec:
               type: string
             addressSpaceType:
               type: string
+            infraConfigRef:
```

Comment:
Here we declare an object, but later, if the example plans for instance, it is used as if it were a string.  How does this work?

Reply:
Fixed